src/formio/components/Signature.stories.mdx

```diff
@@ -0,0 +1,82 @@
+import {Canvas, Meta, Story, Props} from '@storybook/addon-docs';
+import {FormioComponent, FormDecorator} from './story-util';
+
+<Meta
+  title="Form.io components/Signature"
+  argTypes={{
+    backgroundColor: {control: 'color'},
+    description: {
+      type: {name: 'string', required: false},
+    },
+    footer: {
+      type: {name: 'string', required: false},
+    },
+    id: {
+      type: {name: 'string'},
+    },
+    label: {
+      type: {name: 'string', required: true},
+    },
+    penColor: {control: 'color'},
+  }}
+  decorators={[FormDecorator]}
+  parameters={{
+    controls: {sort: 'requiredFirst'},
+  }}
+/>
+
+# Signature
+
+A signature field is a special field that allows someone to sign the field with either their finger on a touch enabled device or with the mouse pointer. This signature will be converted into an image and stored with the form submission.
+
+**Footer Label**
+
+You can enter a short instruction in the footer.
+
+## Default
+
+export const Template = ({
+  backgroundColor = '',
+  description = '',
+  footer = '',
+  id = '',
+  label = '',
+  penColor = '',
```

Comment:
while these are properties that can be specified at the formio level, we don't expose them in our own flavour of the form builder, so we should leave them out to not give wrong impressions:

* backgroundColor
* footer
* id
* penColor

These args (and their argtype definitions) should be removed.